scss/_variables.scss

```diff
@@ -1758,7 +1758,7 @@ $list-group-action-active-bg:       $list-group-active-bg !default; // Boosted m
 // Image thumbnails
 
 // scss-docs-start thumbnail-variables
-$thumbnail-padding:                 $spacer * .5 !default;
+$thumbnail-padding:                 $spacer !default; // Boosted mod
```

Comment:
Why do we change the padding here? Is there a ODS guideline about it?

Rendering seems weird to me in Examples with this value:

![Capture d’écran 2022-08-29 à 08 52 05](https://user-images.githubusercontent.com/17381666/187140832-321f759f-76a8-406f-973b-b9032e6330ee.png)

Reply:
Ah yes, you're right, I haven't seen those. For what I remember the design comment on thumbnails was: 2px border and 20px adjustable padding. So I guess that we could stay with the previous value. Would you be fine with this ? 😄

Reply:
I let you show that to the designers in order to find a rule we agree on.